sdk/typescript/src/types/validator.ts

```diff
@@ -299,7 +301,9 @@ export const MoveActiveValidator = object({
 export const MoveValidatorsFieldsClass = object({
   active_validators: array(MoveActiveValidator),
   next_epoch_validators: array(MoveNextEpochValidator),
-  pending_delegation_switches: ValidatorReportRecords,
+  pending_delegation_switches: optional(
+    object({ contents: array(ValidatorPair) }),
+  ),
```

Comment:
```suggestion
  // TODO: Remove this after 0.28.0 is released
  pending_delegation_switches: optional(ValidatorReportRecords),
```
I think if you change to this, the e2e test should pass. Note that this is the Move definition, not the Rust one

Reply:
Ah I was trying to fix this because it shouldn't be `ValidatorReportRecords`. But I guess it doesn't matter since we are removing it anyway. I'll do the change you suggested and get the tests pass.